build-lib/testing.js

```diff
@@ -51,14 +51,20 @@ function testsTree(pathConfig) {
   ]);
 
   const testModules = babelTranspiler(mergeTrees([testJs, linterTests]), babelConfig(`${pkg.name}/tests`, 'amd'));
+  const testsCommon = babelTranspiler(testJs, babelConfig(pkg.name, 'commonjs'));
```

Comment:
Why not just make this the same as the line above, but in commonjs instead of amd? Why are you excluding the name spacing for test modules? Why skip the linter?